src/components/tabpanel/tabpanel-styles.jsx

```diff
@@ -0,0 +1,9 @@
+export default () => ({
+  root: {
+    padding: [20, 0],
+
+    '&[hidden]': {
```

Comment:
how does this selector work ?

Reply:
this selector says if root element has the attribute hidden="true" or just hidden then these rules should be applied